save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt

```diff
@@ -225,10 +225,15 @@ class SandboxView : AbstractView<Props, SandboxViewState>(true) {
         div {
             className = ClassName("col-6")
             fileUploaderForSandbox(
-                state.files
-            ) { selectedFiles ->
+                state.files,
+                { fileToAdd ->
+                    setState {
+                        files = files.toMutableList().apply { add(fileToAdd) }.toList()
```

Comment:
Will simple `files + fileToAdd` work here?

Reply:
Seems like it will. Will do it

Reply:
Done